plugins/@grouparoo/app-templates/public/templates/table-source/{{{id}}}.js.template

```diff
@@ -42,5 +47,26 @@ exports.default = async function buildConfig() {
       //   },
       // },
     },
+
+    /**
+     * ---------------
+     * Schedule
+     * ---------------
+     *
+     * A Schedule tells Grouparoo how frequently to check the Source for updated
+     * data and import it into the application database. If you would like to
+     * set a Schedule for this Source, uncomment the following section.
+     */
+    // {
+    //   id: {{{schedule_id}}},
+    //   name: {{{schedule_id}}},
+    //   class: "schedule",
+    //   sourceId: {{{id}}}, // The ID of the Source above
+    //   recurring: true, // should this Schedule regularly run?
+    //   recurringFrequency: 1000 * 60 * 15, // 15 minutes, in ms
+    //   options: {
+    //     column: "updated_at", // the column to check for new records in table which this Schedule's Source is using
+    //   },
+    // },
```

Comment:
I think commenting these out for now is the right move.  We'll do something interesting in https://github.com/grouparoo/grouparoo/pull/1313